src/components/select/select.js

```diff
@@ -1136,11 +1142,11 @@ function SelectProvider($$interimElementProvider) {
            */
           function mouseOnScrollbar() {
             var clickOnScrollbar = false;
-            if (ev && (ev.currentTarget.children.length > 0)) {
+            if (mouseDownXPosition != null && ev && (ev.currentTarget.children.length > 0)) {
```

Comment:
@FrankMeyer - While that feature may be more complex and require significant regression issues, my intuition suggests that it may be a better long term solution.